unicorn complains about indexOf(), but includes() is not allowed in ES6

In code that uses `wikimedia/client-es6`, if I use `.indexOf()` to check if something is in an array, I get the following error:
```
/home/catrope/git/codex/packages/vue-components/src/utils.ts
  42:42  error  Use `.includes()`, rather than `.indexOf()`, when checking for existence  unicorn/prefer-includes
```

But if I change it to `includes()`, I get another error instead:
```
/home/catrope/git/codex/packages/vue-components/src/utils.ts
  42:3  error  Unsupported method Array.prototype.includes requires ES2016  no-restricted-syntax
```
The latter error is correct: `includes()` was introduced in ES2016, and `client-es6` requires ES2015. Unicorn shouldn't try to make me use things because they're preferred in environments where I'm not allowed to use them.

--- a/language/rules-es2016.json
+++ b/language/rules-es2016.json
@@ -1,3 +1,7 @@
 {
-	"extends": "./rules-es6"
+	"extends": "./rules-es6",
+	"plugins": [ "unicorn" ],
+	"rules": [
+		"unicorn/prefer-includes": "error"
+	]
 }

--- a/language/rules-es2016.json
+++ b/language/rules-es2016.json
@@ -1,7 +1,7 @@
 {
 	"extends": "./rules-es6",
 	"plugins": [ "unicorn" ],
-	"rules": [
+	"rules": {
 		"unicorn/prefer-includes": "error"
-	]
+	}
 }

--- a/language/rules-es6.json
+++ b/language/rules-es6.json
@@ -14,7 +14,6 @@
 		"no-var": "error",
 		"one-var": "off",
 		"vars-on-top": "off",
-		"prefer-const": "error",
-		"unicorn/prefer-includes": "error"
+		"prefer-const": "error"
 	}
 }

